css-two-column-layout/index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Challenge - Two Column Layout</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <div class="container">
+    <div class="row">
+      <div class="column-full">
+        <div class="top-text">
+          <h1>SCALE, SPEED, AGILITY</h1>
+          <p class="caption">We've dedicated thousands of man hours to making healthcare institutions in the Helium network run 10x more efficiently and deliver an unparalleled quality of care.</p>
+        </div>
+      </div>
+    </div>
+
+    <div class="row">
+      <div class="column-half">
+        <img src="./images/software.svg" alt="software">
+      </div>
+      <div class="text">
+        <h3>INTUITIVE, SMART SOFTWARE</h3>
+        <p>We build products that transform how healthcare facilities run across Africa- making them radically more efficient operationally and financially, a complete turnaround.</p>
+      </div>
+      <div class="column-half">
+        <img src="./images/practice.svg" alt="practice">
+      </div>
+      <div class="text">
+        <h3>PRACTICE MEDICINE, NOT ADMINISTRATION</h3>
+        <p>Our all-in-one electronic health records (EHR), hospital management (HMIS) &amp; billing system helps streamline your facility, reduce waste and improve quality of care.</p>
+      </div>
+    </div>
+
+
+    <div class="row">
+      <div class="column-half">
+        <img src="./images/circumstance.svg" alt="circumstance">
+      </div>
+      <div class="text">
+        <h3>FOR ANY CIRCUMSTANCE</h3>
+        <p>Our intuitive system is built around the peculiarities of your facility; we support offline deployments, sync to the
+        cloud and run on any device 24/7.</p>
+      </div>
+      <div class="column-half">
+        <img src="./images/data.svg" alt="data">
+      </div>
+      <div class="text">
+        <h3>DATA THAT TRANSFORMS</h3>
+        <p>We understand your facility better than you do: we provide visually-appealing real-time analaytics, reports and trends at your fingertips.</p>
+      </div>
```

Comment:
Each `row` you have has two `column-half` elements and two `text` elements set to `width: 100%`. That means each row has 300% width crammed into it. This needs to be better organized with accurate measurements applied to each item. Don't let flexbox fix your messes for you 🤢

Reply:
Also, `text` should be a `column` class, having width attributed to semantic classes when you have a layout class system is usually bad news.